metasv.py

```diff
@@ -169,14 +189,21 @@
     tool_out_fd.close()
     pysam.tabix_index(tool_out, force=True, preset="vcf")
 
+# Do merging here
 for sv_type in sv_types:
   logger.info("Processing SVs of type %s" % (sv_type))
   tool_merged_intervals[sv_type] = []
+
+  # Do the intra-tool merging
   for tool in tools:
     if sv_type not in intervals[tool]: continue
     logger.info("First level merging for %s for tool %s" % (sv_type, tool))
     tool_merged_intervals[sv_type] += sv_interval.merge_intervals(intervals[tool][sv_type])
+
+  # Do the inter-tool merging
   merged_intervals = sv_interval.merge_intervals(tool_merged_intervals[sv_type])
+
+  # Marghoob.... what does this do? :/
```

Comment:
This merges all the calls. Later on, this merged set is used to generate a merged set where the contributing intervals are a significant fraction of the merged interval.